docs/developer/decision-records/2022-06-08-identity-hub/README.md

```diff
@@ -23,6 +23,8 @@ Each participant in the dataspace will have a single [decentralized identifier](
 
 During participant-to-participant communication via IDS REST, the request destination participant queries the Verifiable Presentations of the request originator participant. Access to resources is granted or denied according to the policies in place and the available Verifiable Presentations.
 
+Verifiable Presentations are issued and signed by a trusted authority. Before applying any policies, participants access the public key of that trusted authority from its DID Document, and check that Verifiable Presentations are valid.
+
```

Comment:
what happens if they're not valid? that would certainly be interesting :)

Reply:
It is certainly interesting, but not sure if this document is the right place to go deep into much more detail. I have added 2 lines on that topic, let me know what you think.